src/views/MapView.js

```diff
@@ -59,6 +62,50 @@ const geojsonMarkerOptionsGray = {
   fillOpacity: 0.8,
 };
 
+// When comparing routeline and navigationline the cross sign is used instead of circle
+const crossIconRed = L.divIcon({
+  className: "material-icon",
+  html: ReactDOMServer.renderToString(
```

Comment:
Dokumentaation mukaan ReactDOMServer.renderToString:iä kuuluisi käyttää vain palvelimen puolella, eli ei frontendissä. Onko jotain toista tapaa luoda leaflet-ikoni?

Reply:
Näyttää siltä että itseasiassa tuota kyseistä funktiota voi käyttää myös clientin puolella, vaikka dokumentaatiossa sanottiin vähän toisin: https://stackoverflow.com/questions/37079847/is-it-ok-to-use-reactdomserver-rendertostring-in-the-browser-in-areas-where-reac
Ja näytti myös että ei ole kovin hyviä vaihtoehtoja myöskään. Eli sanoisin tuon perusteella että tämä on ok.